packages/server/lib/controllers/v1/account/managed/getCallback.ts

```diff
@@ -101,7 +101,12 @@ export const getManagedCallback = asyncWrapper<GetManagedCallback>(async (req, r
         }
 
         // Create a user
-        user = await userService.createUser(authorizedUser.email, name, '', '', accountId);
+        user = await userService.createUser({
+            email: authorizedUser.email,
+            name,
+            account_id: accountId,
+            email_verified: true
```

Comment:
I guess this line is the fix

Reply:
no actually it was true by default, the actual fix is on signup as it was set as false